griptape/artifacts/boolean_artifact.py

```diff
@@ -10,11 +10,13 @@ class BooleanArtifact(BaseArtifact):
     meta: dict[str, Any] = field(factory=dict, kw_only=True, metadata={"serializable": True})
 
     @classmethod
-    def parse_bool(cls, value: Union[str, bool]) -> BooleanArtifact:
+    def parse_bool(cls, value: Union[str, bool, TextArtifact]) -> BooleanArtifact:
```

Comment:
Should be named `from_bool` for consistency with other similar methods.

Reply:
- a bool is one of the possible inputs, so `from_bool` isn't really accurate.
- i wanted to convey the fact that its going to actually evaluate the literal text in the string, rather than just coerce the input type to a bool type.